### DISCUSSION TITLE
Different settings for different heading levels?

### DISCUSSION THREAD

**Discussion by einarpersson** [MeanderingProgrammer/render-markdown.nvim#240] [Q&A] [answered]
I want h1 and h2 to stand out more than h3, h4 etc.

I have border and padding enabled for headings but would like to if possible remove it for the h3, h4 and beyond. It does not look like this is possible atm, is it the case?

**Comment by MeanderingProgrammer** [comment, marked as answer]
This was supported for the padding configurations but not the border, added that here: https://github.com/MeanderingProgrammer/render-markdown.nvim/commit/c83fc5694cd19c576af188f5f3313513bf467272.

The border can now be a single value like before, or a list of values where the last value in the list is used for the remaining headings.

So if you want only h1 & h2 to have borders but not h3 -> h6 you would do the following:

```lua
require('render-markdown').setup({
    heading = {
        border = { true, true, false },
    },
})
```

You can do the same thing for the `width`, `left_pad`, `right_pad`, `left_margin` and `min_width` configurations.
